### PR TITLE
Added support for setting lower/upper limit for zoom/pan range

### PR DESCRIPTION
**Pull request by bjornnorman**
I added two functions for enforcing the limits and ensured they are called by all zoom/pan functions.
To support both x and y directions, I supplied the detected direction with the options object before passing it along.
The solutions works excellently in my case with a horizontal timescale graph, but I haven't tested it with other scale types, but in theory it should handle that too as it is.
The README.MD file was updated to include the new range settings.
Hope you find this useful - let me know if you have any questions :)

### CODE REVIEW

**Comment by bjornnorman** [comment]
Damnit. Didn't see you already had another pull request for this feature...
Very well, now you'll have to decide which is the more elegant! ;)

**Comment by compwright** [comment]
Resolves #37

File: src/chart.zoom.js [hunk 1]
```diff
@@ -42,6 +42,26 @@ function directionEnabled(mode, dir) {
...
+		var rangeMax = zoomPanOptions.rangeMax[zoomPanOptions.scaleAxes];
+        if (newMax > rangeMax) {
+            newMax = rangeMax;
+        }
+    }
+    return newMax;
+}
+
+function rangeMinLimiter(zoomPanOptions, newMin) {
+    if (zoomPanOptions.scaleAxes && zoomPanOptions.rangeMax && zoomPanOptions.rangeMax[zoomPanOptions.scaleAxes]) {
```
Comment (compwright) [inline src/chart.zoom.js]: Appears to be typos: should it be `zoomPanOptions.rangeMin`?

Comment (bjornnorman) [inline src/chart.zoom.js]: Good catch - you're right, I missed that line when I cut and pasted the Max function ;) I've updated the pull request now.

**Comment by ericnkatz** [comment]
Is there a working example of this on Codepen somewhere? I cannot seem to get this to work.

**Comment by ericnkatz** [comment]
@bjornnorman @compwright the limits don't work when values are 0 due to type coercion?

**Comment by bjornnorman** [comment]
Yes, there should probably be added a "=== null" test to avoid ignoring "0" limits, and a default config value of "null" in case the max/min params aren't specified.

**Comment by sridhark158** [comment]
Is there a working example of this codepen/jsfiddle ?

**Comment by arnab333** [comment]
Is there any working example for "rangeMin" and "rangeMix"?

**Pull request merged**